modules/nf-core/last/mafconvert/main.nf

```diff
@@ -10,7 +10,7 @@ process LAST_MAFCONVERT {
     input:
     tuple val(meta), path(maf)
     val(format)
-    path(fasta)
+    tuple path(fasta), path(fai), path(gzi)
```

Comment:
```suggestion
    tuple val(meta2), path(fasta),
    tuple val(meta3), path(fai),
    tuple val(meta4), path(gzi)
```
To keep the reference inputs consistent with other nf-core modules

Reply:
Thanks, I did not pay attention that `samtools/faidx` was providing the files in separate output channels.